### PR TITLE
[RDF][Doc] Add print to df106

### PR DESCRIPTION
**Pull request by stwunsch**
Streamlines the behaviour with respect to df105 and df107

### CODE REVIEW

File: tutorials/dataframe/df106_HiggsToFourLeptons.py [hunk 1]
```diff
@@ -193,4 +193,5 @@ def merge_histos(label):
 text.DrawLatex(0.21, 0.80, "#sqrt{s} = 13 TeV, 10 fb^{-1}")
 
 # Save the plot
-c.SaveAs("HiggsToFourLeptons.pdf")
+c.SaveAs("df106_HiggsToFourLeptons.png")
+print("Save figure to df106_HiggsToFourLeptons.png")
```
Comment (eguiraud) [inline tutorials/dataframe/df106_HiggsToFourLeptons.py:197]:
```suggestion
print("Saved figure to df106_HiggsToFourLeptons.png")
```
?

**Review by eguiraud — APPROVED** [review]
LGTM

File: tutorials/dataframe/df106_HiggsToFourLeptons.py (hunk 1 above)
Comment (stwunsch) [reply tutorials/dataframe/df106_HiggsToFourLeptons.py]: Yep, true. I've fixed it for all tutorials in one go!

**Comment by eguiraud** [comment]
ah can you use `[DF]` instead of `[RDF]` in the commit message please? :smile: (yes we have been inconsistent in the past but `[DF]` is the prevalent spelling)

**Comment by stwunsch** [comment]
> ah can you use `[DF]` instead of `[RDF]` in the commit message please?  (yes we have been inconsistent in the past but `[DF]` is the prevalent spelling)

Done!

**Pull request merged**